docs/reference/modules/discovery/gce.asciidoc

```diff
@@ -1,6 +0,0 @@
-[[modules-discovery-gce]]
```

Comment:
Ditto adding to https://www.elastic.co/guide/en/elasticsearch/reference/master/redirects.html

Reply:
TIL, thanks. Done in 400b2e46d68.